easybuild/tools/module_naming_scheme/mns.py

```diff
@@ -164,3 +164,11 @@ def is_short_modname_for(self, short_modname, name):
                        short_modname, name, modname_regex.pattern, res)
 
         return res
+
+    def det_generate_devel_module(self):
+        """
+        Determine if a devel module should be generated. Can be used to create a separate set of modules with a different naming scheme.
```

Comment:
line too long (> 120 chars), easy/sensible fix is to move 2nd sentence to new line